pybossa/cache/helpers.py

```diff
@@ -29,6 +29,17 @@
 
 session = db.slave_session
 
+def n_gold_tasks(project_id):
+    """Returh the number of gold tasks for a given project"""
+    query = text('''SELECT COUNT(*) AS n_gold_tasks FROM task
+                WHERE project_id=:project_id
+                AND state !='enrich'
+                AND calibration = 1;''')
+    result = session.execute(query, dict(project_id=project_id))
+    n_gold_tasks = 0
```

Comment:
this could be total_gold_tasks to avoid name conflict w/ func name

Reply:
thanks for pointing out, will fix.